knimin/handlers/pm_sequence.py

```diff
@@ -0,0 +1,43 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from collections import defaultdict
+
+from tornado.web import authenticated
+
+from knimin.handlers.base import BaseHandler
+from knimin.handlers.access_decorators import set_access
+from knimin import db
+
+
+@set_access(['Admin'])
+class PMSequenceHandler(BaseHandler):
```

Comment:
[Here](https://github.com/biocore/labadmin/pull/203/files#diff-f41cc3c10712186becef17efacc15f7fR16) class name was wrong, I fixed it